remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingClient.java

```diff
@@ -707,5 +711,25 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws E
                 NettyRemotingClient.this.putNettyEvent(new NettyEvent(NettyEventType.EXCEPTION, remoteAddress, ctx.channel()));
             }
         }
+
+        @Override
+        public void channelInactive(ChannelHandlerContext ctx) throws Exception {
```

Comment:
Can channelInactive represent broker crash?

Reply:
Usually, it does work